helm_deploy/hmpps-workload/templates/dlq-cronjob.yaml

```diff
@@ -12,6 +12,9 @@ spec:
     spec:
       template:
         spec:
+          {{- if .Values.serviceAccountName }}
```

Comment:
if you wanted this to work if would need to be:
```
{{- if .Values.generic-service.serviceAccountName }}
```
However I dont think adding a service account will fix the issue you have.